src/pages/Crowdloan.vue

```diff
@@ -39,7 +39,7 @@ export default defineComponent({
   },
   setup() {
     const startTime = moment.utc([2021, 1, 1]);
-    const endTime = moment.utc([2021, 11, 17]).add(5, 'h').add(56, 'm');
+    const endTime = moment.utc('2021-12-02').add(15, 'h');
```

Comment:
Keep in mind that the auction will end at block number `7885710`. Because block times are not consistent, UTC will not be accurate.

https://polkadot.subscan.io/extrinsic/7860510-0?event=7860510-0